src/unix.rs

```diff
@@ -0,0 +1,8 @@
+
+/// Unix specific extension methods.
```

Comment:
I don't know how to properly link std docs, does the link to this trait may break in the future if change occur in the std ?